plugin/core/clients.py

```diff
@@ -23,9 +23,8 @@
 clients_by_window = {}  # type: Dict[int, Dict[str, Client]]
 
 
-class LspTextCommand(TextCommand):
-    # Passing capability='' due Sublime Text bug: https://github.com/SublimeTextIssues/Core/issues/2175
-    def __init__(self, view, capability='', last_check=lambda: True):
+class LspTextCommand(object):
```

Comment:
Would it not make more sense to rename this to something like `AutoDisableMixin` ?

Reply:
I deprecated the mixin because it is requires to write more code then usual, on the last commit https://github.com/evandroforks/LSP/commit/329038ab04d360302146c166036a9548ddcb5ceb you can see how much code I removed.

Also, would be required to add an exception for `mypy`
1. https://github.com/SublimeTextIssues/Core/issues/2066#issuecomment-362456473
1. https://travis-ci.org/tomv564/LSP/builds/336313027?utm_source=github_status&utm_medium=notification since it is failing with this:
```
mypy -p plugin
plugin/core/clients.py:26: error: Too many arguments for "__init__" of "object"
```